### DISCUSSION TITLE
Low alerts doesn't trigger phone alarm

### DISCUSSION THREAD

**Discussion by Hexonium** [NightscoutFoundation/xDrip#3412] [Bug reports]
I noticed during last night that low alerts were not triggering the alarm on my phone anymore (the logs show that they were snoozed after 9s or 10s, which I'm sure is not an action I took). My phone was in do not disturb mode but xdrip+ is in the exception list for notifications (and the alarms were working fine the days before). After disabling the do not disturb mode, I started receiving alarms again. 

I haven't change any parameter on my phone recently. I'm using a Pixel 5a with Android 14 (build AP1A.240305.019.A1) and xdrip version efb3322-2024.03.27

**Comment by Navid200** [comment]
Have a look at this:
https://navid200.github.io/xDrip/docs/Silent-alert.html
